tools/crc32.c

```diff
@@ -0,0 +1,119 @@
+/* crc32 -- calculate and print POSIX checksums for a specific string.
+
+   This tool is a stipped clone of cksum tool from Linux coreutils.
+   Is using the same crc32 lookup table crctab as coreutils.
+   Is necessary for ifupdown-ng when the cksum tools is not avaiable on
+   a system and we need VRRP name generation.
+
+   Copyright (C) 2023 EasyNetDev.
+
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
```

Comment:
As above.

Reply:
Please check my comment above and add your comments.